views/partials/jira-configuration-error-summary.hbs

```diff
@@ -10,25 +10,27 @@
         You could continue to use the integration and still view the backfilled data in Jira, while the app attempts to reprocess these entities.</p>
     </div>
 
-    <div class="error-summary-border-bottom">
-      <div class="error-summary-errors-header error-summary-border-bottom">
-        Description
-      </div>
+    <div class="error-summary-border-top">
+      <h4 class="error-summary-errors-header">
+        Please resolve the following errors:
+      </h4>
+      <ul>
       {{#if syncErrors.PERMISSIONS_ERROR}}
-      <div>
+      <li>
         <b>{{syncErrors.PERMISSIONS_ERROR}}</b> tasks have not been backfilled. Please
```

Comment:
nit: If the count here is 1 can we apply some single logic to change tasks to task? 1 tasks reads funny.

Reply:
will do!